l5kit/l5kit/dataset/ego.py

```diff
@@ -98,32 +100,45 @@ def get_frame(self, scene_index: int, state_index: int, track_id: Optional[int]
             )
         data = self.sample_function(state_index, frames, self.dataset.agents, tl_faces, track_id)
 
-        target_positions = np.array(data["target_positions"], dtype=np.float32)
-        target_yaws = np.array(data["target_yaws"], dtype=np.float32)
-
+        # [history_num_frames + 1, 2]
         history_positions = np.array(data["history_positions"], dtype=np.float32)
+        # [history_num_frames + 1, 1]
         history_yaws = np.array(data["history_yaws"], dtype=np.float32)
+        # [future_num_frames + 1, 2]
+        history_velocities = np.array(data["history_velocities"], dtype=np.float32)
+
+        # [future_num_frames, 2]
+        target_positions = np.array(data["target_positions"], dtype=np.float32)
+        # [future_num_frames, 1]
+        target_yaws = np.array(data["target_yaws"], dtype=np.float32)
+        # [future_num_frames, 2]
+        target_velocities = np.array(data["target_velocities"], dtype=np.float32)
 
+        host_id = (self.dataset.scenes[scene_index]["host"],)
```

Comment:
you can also remove the tuple. Torch will give you a list of strings.

What I'm worried about is what will happen when you try to move the entire dict in cuda though

Reply:
hmm, it is caused by autoformatting, addressed